docs/jazzy.yml

```diff
@@ -22,68 +22,108 @@ custom_categories:
       - NavigationViewController
       - NavigationViewControllerDelegate
       - RouteVoiceController
+      - VoiceControllerDelegate
+      - MapboxVoiceController
+      - NavigationOptions
+      - NavigationView
```

Comment:
I tried to group items to where they belong, but I might be mistaken :)

Reply:
This is a good place to put these types. 👍